tests/server/routes/members-only-test.js

```diff
@@ -7,7 +7,7 @@ var db = require('../../../server/db');
 
 var supertest = require('supertest');
 
-describe('Members Route', function () {
+xdescribe('Members Route', function () {
```

Comment:
If you don't need this test suite any more, you might want to just delete it, not mark it as pending. @noeladd